containers/PriceFeed.ts

```diff
@@ -0,0 +1,35 @@
+import { createContainer } from "unstated-next";
+import { useState, useEffect } from "react";
+import { BigNumber } from "ethers";
+
+import { getOffchainPrice } from "../utils/getOffchainPrice";
+
+import Token from "./Token";
+
+function usePriceFeed() {
+  const { symbol: tokenSymbol } = Token.useContainer();
+
+  const [latestPrice, setLatestPrice] = useState<BigNumber | null>(null);
+
+  const queryPrice = async () => {
+    if (tokenSymbol) {
+      const query = await getOffchainPrice(
+        tokenSymbol.includes("yCOMP") ? "compusd" : "ethbtc"
```

Comment:
This hard-coded logic is meant to serve as a temporary placeholder for now. The purpose of this container is to house all logic for fetching off-chain prices via API